### DISCUSSION TITLE
***NOW WORKING*** vdh 9.0.2.12 cannot download from twitch

### DISCUSSION THREAD

**Discussion by romelussaladar** [aclap-dev/video-downloadhelper#959] [Help and Q&A]
### Your question

Using CHROME Version 126.0.6478.127 (Official Build) (64-bit) and VDH ver 9.0.2.12 and CANNOT detect STREAM like PREVIOUSLY to have CHOICE to DOWNLOAD stream or CLIPS.

### Addon & CoApp version and details

Something has changed within the last 5 days. I was able to DOWNLOAD last clip on twitCH on 25Jul24. Today NOW VDH cannot DETECT stream at all, therefore NO DOWNLOAD choices. I have attempted MULTIPLE TWITCH CHANNELS, just to see if VDH can DETECT ANY stream, but NO nothing.  I also CLOSED CHROME and OPENED again, and STILL cannot detect any stream on TWITCH using CURRENT VDH version. 9.0.2.12.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3810, Firefox 127.0.2 64-bit, licensed VDH 9.0.2.12a1 beta, CoApp 2.0.19.  You really should provide all of that information with any problem you report.

I just now went to the Twitch home page & told VDH to record the random livestream that was there as the default at the top of the page.  I let it run for 1 minute then stopped it.  It gave me a 1920x1080 MP4 that played just fine in VLC with both video & audio.

That doesn't help you on Chrome but maybe it helps @paulrouget Paul narrow things down.  Also, it might be worth it for you to try the beta VDH for Chrome.  Maybe there's something in there that fixed a relevant bug.  You never know until you try it.

Other users have had issues with either or both of their antivirus & their firewall interfering with VDH.  There are several discussions of the topic in this forum that you can search for & read.  Perhaps that is worth eliminating as a possibility.  I'm running just the Windows Defender that comes built in with Windows 11.  There was another user recently who discovered that he actually had a second antivirus installed that he thought he had uninstalled.  Software is always stealth-installing other stuff.  I just yesterday got a new version of CCleaner & I had to be sure I told it no, I do NOT want AVG antivirus, just CCleaner.  At least the prompt was clear & obvious.  Other software might not be so ethical.  So don't dismiss this possibility out of hand.  Check this possibility.

**Reply by romelussaladar** [reply]
Hi @GrampaWildWilly ,

Yeah I am using Windows 11 Home Ver 23H2 as well and CHROME Version 126.0.6478.127 (Official Build) (64-bit) and VDH ver 9.0.2.12. I don't have any antivirus at all either. I have been using Twitch for a long time, and in PREVIOUS posts from other OLDER issues with VDH, I have encountered the same issue of not able to GRAB CURRENT TWITCH Livestreams for now. BUT... I can grab CLIPS (short 1 minute and half short videos clipped from the stream), which I press PLAY and VDH detects that video clip.  I have attempted to TURN ON VDH first, THEN go to the twitch livestream (any user) and then VDH cannot detect any ACTIVE stream, even though the stream is active on the twitch page. I have turned OFF VDH and turned it on again, and REFRESHED the twitch page livestream, to see if VDH would detect the stream, but again no VDH did not change colour to show it can detect the stream, so there was no option to download the current twitch stream. Only the clipped stream, VDH can detect that VOD, so I can then choose the VDH options to download as usual. @paulrouget  hopefully this gives you the current observations with VDH ver 9.0.2.12 for now.

**Reply by GrampaWildWilly** [reply]
I suggested above that you install the beta version of VDH.  You should try that.  It might fix your problem.  Like I said, it works for me on Firefox so this appears to be a problem specific to Chrome.

**Reply by GrampaWildWilly** [reply]
You should also follow these instructions:

https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

Immediately after following those instructions, compose a post here & do Shift+Insert.  Then post what gets pasted.

**Reply by romelussaladar** [reply]
> You should also follow these instructions:
> 
> https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
> 
> Immediately after following those instructions, compose a post here & do Shift+Insert. Then post what gets pasted.

![Screenshot 2024-07-11 203455](https://github.com/aclap-dev/video-downloadhelper/assets/111546622/865d1c7c-bdaf-4df7-8a70-c6d2a710a958)
![Screenshot 2024-07-11 205447](https://github.com/aclap-dev/video-downloadhelper/assets/111546622/1da5788b-0ac7-4f37-8aa7-2da56712f761)

Hi @GrampaWildWilly , SEE above screen shots showing CHROME with VDH ver 9.0.2.12  NOT DETECTING Twitch livestream. All Chrome info as mentioned earlier, for windows 11. I think you are CORRECT with my current problem showing CHROME having the issue, as opposed to working for your Firefox browser. When I CLIP livestreams and twitch creates 1min 30second videos, that clip when I press PLAY, VDH can detect that for download. HOPE this helps @paulrouget and the team figure what can be fixed for at least Chrome.

Cheers.

**Reply by GrampaWildWilly** [reply]
That is all well & good.  But I suggested you try using the beta version of VDH for Chrome.  You should try that.  Maybe it will fix your issue.  Also . . .

![#01](https://github.com/user-attachments/assets/4f4b1f5a-3efb-4c64-a1f2-9e24b7275a96)

Then immediately after clicking that button, in a post here, type Shift+Insert.  Then post what gets pasted.

**Reply by GrampaWildWilly** [reply]
Here.  I'll go first.

version: 9.0.2.14a1
target: mozilla
channel: beta
lang: en-US
coapp: {"found":true,"path":"C:\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"&lt;removed&gt;","key":"&lt;removed&gt;"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "M:\\VDH Testing",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "M:\\VDH Testing"
        }
    },
    "concurrent_downloads_max": 32,
    "show_success_notification_for_icognito": true,
    "theme": "system",
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": true,
        "hide_downloaded": false
    },
    "default_action": "download_as",
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 5,
    "successfull_dl": 33
}

Now you do it.

**Reply by romelussaladar** [reply]
OK here is what I got from your instructions....


version: 9.0.2.12
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"*****************"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\rsala\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\rsala\\Downloads"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 1,
    "successfull_dl": 90
}

**Reply by GrampaWildWilly** [reply]
Excellent.  That needs to be part of any problem you report in future.

However, it seems there is a solution.  See below in the suggestion from Matt.

**Comment by NEELSONO** [comment]
Hi.

Please help me; I have the same problem. But I'm sure it came from CCleaner and/or Wise Registry Cleaner. 
After the application, I can't download anything on Twitch anymore! I have other PCs, and there it still works. 

I avoid the cleaner now, although I want to use it. 

The problem is not due to the Chrome version.

Does anyone have an idea of the new findings?

Thank you very much!

**Reply by GrampaWildWilly** [reply]
Is anybody ever going to take my advice and install the beta version of VDH for Chrome?

I have used CCleaner countless times over the years, the most recent being just a couple of weeks ago in preparation for taking my periodic system & data backups.  It has never had an impact on VDH.  Why do you believe there's a connection there?  Let's see some evidence.  Otherwise, I would suggest you are seeing a coincidence, that your use of CCleaner just happened to occur at the same time an unrelated issue occurred that affected Twitch.  I don't think you should suddenly become afraid to use your registry cleaners.

**Comment by NEELSONO** [comment]
Ok, I used it on 5 PCs and it stopped working on all of them. If it can't be CCleaner, then it must be Wise Registry Cleaner. 
Could this be the problem?

**Reply by GrampaWildWilly** [reply]
I've never used that product.  In fact, I'd never heard of it before you mentioned it.  When you run it, does it show you what it's going to clean before it cleans it?  Was there anything that looked related to VDH that was going to get removed?  For future reference, you probably want to tell it not to clean any items that look related to VDH.

I think we're looking at maybe VDH needing to be reinstalled.  On one of your computers -- let's see if we can get one of them back on the air before we do them all -- uninstall VDH, then install the BETA for VDH for Chrome.  THE BETA.  I keep mentioning the BETA & nobody seems to hear me.  Install the BETA.  Then see if the Twitch problem goes away.  If you're looking for the beta, just click the Wiki link at the top of any page of this forum.  You'll find a reference to the beta versions easily enough.  Click that reference, then click the proper link for installing the beta for VDH for Chrome.

**Reply by NEELSONO** [reply]
I now have the Chrome BETA version. It doesn't work either. 😓

**Reply by Mattmjs** [reply]
Try in Chromium the open source version of Chrome and see if it works.

**Reply by NEELSONO** [reply]
This does not appear to be a normal browser. It's too complicated for me, I don't know how it works.

**Reply by Mattmjs** [reply]
It's not complicated, pick the first stable build here : https://chromium.woolyss.com/
You can install it or use the portable build. Then add the extension. See if you get detection from Twitch. If it works in Chromium then there must be a bug in Chrome. I use the Ungoogled Chromium build and I have detection

**Reply by NEELSONO** [reply]
It works my friend! Many thanks 😃

**Reply by Mattmjs** [reply]
Sounds like a bug in the Chrome code occurred somewhere around version 126. where it is omitting the stream information
in the network. This bug struck the Firefox browser a year ago before Mozilla fixed it.
Does Chrome have a bug reporting site ? , if so they would need to know about the problem.

**Reply by NEELSONO** [reply]
Well, I don't think it's Chrome itself. After I use CCleaner/Wise Registry Cleaner it no longer works with Twitch. Something is probably being destroyed in Chrome. With Chromium works finally.

**Reply by Mattmjs** [reply]
I use Firefox mainly but I have Ungoogled Chromium for testing purposes. Do you think it could work in Chrome if those
programs were not used. I assumed it might be a Chrome bug, maybe it isn't. Years ago I used CCleaner, not anymore though
and I don't use any registry cleaners. Windows 10 has it's own clean up utility and I manually clear cache history etc..
in Firefox.

**Comment by paulrouget** [comment]
Please don't use caps: https://github.com/aclap-dev/video-downloadhelper/wiki/Code-Of-Conduct

I'll look into this. Seem serious…

**Comment by paulrouget** [comment]
> Ok, I used it on 5 PCs and it stopped working on all of them. If it can't be CCleaner, then it must be Wise Registry Cleaner.

Yeah.

If you used those, it's likely that the registry got cleaned out. We write the location of the coapp in the registry.

Basically, the solution would have been to re-install the coapp.

**Comment by romelussaladar** [comment]
I updated Chrome, and now everything works. @paulrouget I am guessing it was what @GrampaWildWilly mentioned regarding Chrome at that time having something interfering with VDH. I can now grab TWITCH videos live, and the new features work great, choosing the different resolution sizes.

Just an update
